src/components/allproperties/AllPropertiesData.jsx

```diff
@@ -1,28 +1,48 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import Navbar from "../Navbar/Navbar";
 import Footer from "../../components/footer/Footer";
 import AllProperties from "./allproperties";
 import "./propertiesdata.css";
 import PropertiesInfo from "./PropertiesInfo";
+import axios from 'axios'
+import { useSelector } from "react-redux";
+
+const url = 'http://localhost:4000/v1/properties'
```

Comment:
This API URL should be in an environment variable, research how to work with the environment variable in React. If the API URL changes, you don't always have to modify the code to reflect the changes.